bitnami/external-dns/values.yaml

```diff
@@ -0,0 +1,166 @@
+## Bitnami etcd image version
```

Comment:
Aren't we using a values-production.yaml as well?

Reply:
I'm not sure if it makes sense in this chart. I mean, I don't know if it makes sense to replicate it.